app/api/og-image/route.ts

```
@@ -0,0 +1,530 @@
+  try {
+    if (domain === 'Bluesky' && url.includes('williamcallahan.com')) {
+      // For Bluesky, try to extract the avatar image from the HTML
+      console.log('🔍 Attempting to extract Bluesky profile avatar from HTML...');
+
+      // Look for avatar image patterns in Bluesky's HTML
+      const avatarPattern = /<img[^>]*class="[^"]*avatar[^"]*"[^>]*src="([^"]+)"/i;
+      const avatarMatch = html.match(avatarPattern);
+
+      if (avatarMatch && avatarMatch[1]) {
+        console.log(`✅ Found Bluesky avatar: ${avatarMatch[1]}`);
+        result.profileImage = avatarMatch[1];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)